lib/utils/strings.ts

```diff
@@ -1,3 +1,6 @@
+import { TFunction } from 'next-i18next';
```

Comment:
I am really curious where or how you found out that "t" is of type "TFunction".

Reply:
In the module itself.In the node_modules folder, there is an i18next folder. Within that folder, in index.d.ts, you'll find type definitions for i18next, including e.g. one for WithT, which takes t of type TFunction as parameter.

Reply:
ahhhh really cool. Thanks :)
Spent quite some time searching for it online.